app/views/games/categories/show.slim

```diff
@@ -1,7 +1,7 @@
 - title @game.name
 - content_for(:header) do
   ol.breadcrumb
-    li.breadcrumb-item = link_to "Games", games_path
+    li.breadcrumb-item = link_to "Games", search_path
```

Comment:
Additional comment below on `config/routes.rb` but if we keep the `/games` route around as a redirect to `/search` until we add the games directory, we can keep using `games_path` in all these places and we won't forget to change it back when the games directory at `/games` is added.

Reply:
Do you want this to just be a simple redirect in the route file like search used to redirect to games?

Reply:
Yeah that's perfect.